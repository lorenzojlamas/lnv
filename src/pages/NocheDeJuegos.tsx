import React, { Component } from 'react';

export class NocheDeJuegos extends Component {

    render() {
        return (
            <div>
                <p>
                    Noche de juegos
                </p>
            </div>
        );
    }
}