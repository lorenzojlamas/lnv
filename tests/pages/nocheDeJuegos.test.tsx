import React from 'react';
import renderer from 'react-test-renderer';
import NocheDeJuegos from '../../pages/nocheDeJuegos';

describe('nocheDeJuegos', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<NocheDeJuegos />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
