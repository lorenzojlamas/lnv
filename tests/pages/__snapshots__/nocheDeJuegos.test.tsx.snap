// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nocheDeJuegos should render correctly 1`] = `
<div
  className="root-container"
>
  <header>
    <nav
      className="navbar navbar-expand-lg navbar-dark bg-dark"
    >
      <a
        className="nav-link"
        href="/"
        onClick={[Function]}
        onKeyDown={[Function]}
      >
        <span
          className="navbar-brand"
        >
          <img
            className="img-fluid"
            src="/logoCompleto.png"
            width="120px"
          />
        </span>
      </a>
      <button
        aria-controls="responsive-navbar-nav"
        aria-label="Toggle navigation"
        className="navbar-toggler collapsed"
        onClick={[Function]}
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        aria-expanded={null}
        className="navbar-collapse collapse"
        id="responsive-navbar-nav"
      >
        <div
          className="mr-auto navbar-nav"
          onKeyDown={[Function]}
        >
          <a
            className="nav-link"
            data-rb-event-key="/discord"
            href="/discord"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Discord
          </a>
          <a
            className="nav-link"
            data-rb-event-key="/exactasJuega"
            href="/exactasJuega"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Exactas juega
          </a>
          <a
            className="nav-link"
            data-rb-event-key="/nocheDeJuegos"
            href="/nocheDeJuegos"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Noche de juegos
          </a>
          <a
            className="nav-link"
            data-rb-event-key="/pabellonF"
            href="/pabellonF"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Podcasts
          </a>
          <a
            className="nav-link"
            data-rb-event-key="/tournament"
            href="/tournament"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Torneos
          </a>
          <a
            className="nav-link"
            data-rb-event-key="/xplora"
            href="/xplora"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Divulgación
          </a>
        </div>
        <div
          className="navbar-nav"
          onKeyDown={[Function]}
        >
          <a
            className="nav-link"
            data-rb-event-key="/about"
            href="/about"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            About
          </a>
        </div>
      </div>
    </nav>
  </header>
  <div
    className="body-pages"
  >
    <div
      className="card bg-secondary text-white"
    >
      <div
        className="card-header"
      >
        <div
          className="content-md-center container-fluid"
          style={
            Object {
              "textAlign": "center",
            }
          }
        >
          <h1>
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-headphones fa-w-16 "
              data-icon="headphones"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M256 32C114.52 32 0 146.496 0 288v48a32 32 0 0 0 17.689 28.622l14.383 7.191C34.083 431.903 83.421 480 144 480h24c13.255 0 24-10.745 24-24V280c0-13.255-10.745-24-24-24h-24c-31.342 0-59.671 12.879-80 33.627V288c0-105.869 86.131-192 192-192s192 86.131 192 192v1.627C427.671 268.879 399.342 256 368 256h-24c-13.255 0-24 10.745-24 24v176c0 13.255 10.745 24 24 24h24c60.579 0 109.917-48.098 111.928-108.187l14.382-7.191A32 32 0 0 0 512 336v-48c0-141.479-114.496-256-256-256z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
             
            <span
              className="badge badge-dark"
            >
              Noche de juegos
            </span>
          </h1>
        </div>
      </div>
      <div
        className="card-body"
      >
        <div
          className="container"
        >
          <p>
            La Noche de Juegos es un evento bimensual donde les estudiantes de exactas nos juntamos a compartir varias horas con diversos juegos. Es una actividad que empezó hace varios años de la mano de la ComCom y la hemos transformado en una propuesta virtual para no perder la magia. Éstos juegos incluyen el codenames, catan, ajedrez, pictonary, among us y muchos otros más.
          </p>
          <p>
            Seguinos en nuestras redes y participá de la próxima Noche de Juegos!
          </p>
          <div
            style={
              Object {
                "alignContent": "center",
                "alignItems": "center",
                "boxSizing": "border-box",
                "display": "flex",
                "flexDirection": "row",
                "flexWrap": "nowrap",
                "justifyContent": "center",
              }
            }
          >
            <a
              className="btn btn-secondary btn-lg"
              href="https://www.instagram.com/ndj.exactas.uba/"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              target="_blank"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-instagram fa-w-14 fa-2x "
                data-icon="instagram"
                data-prefix="fab"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 448 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </a>
            <a
              className="btn btn-secondary btn-lg"
              href="https://www.facebook.com/ndj.exactas.uba"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              target="_blank"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-facebook fa-w-16 fa-2x "
                data-icon="facebook"
                data-prefix="fab"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </a>
            <a
              className="btn btn-secondary btn-lg"
              href="https://discord.gg/TJfv4AF"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              target="_blank"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-discord fa-w-14 fa-2x "
                data-icon="discord"
                data-prefix="fab"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 448 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M297.216 243.2c0 15.616-11.52 28.416-26.112 28.416-14.336 0-26.112-12.8-26.112-28.416s11.52-28.416 26.112-28.416c14.592 0 26.112 12.8 26.112 28.416zm-119.552-28.416c-14.592 0-26.112 12.8-26.112 28.416s11.776 28.416 26.112 28.416c14.592 0 26.112-12.8 26.112-28.416.256-15.616-11.52-28.416-26.112-28.416zM448 52.736V512c-64.494-56.994-43.868-38.128-118.784-107.776l13.568 47.36H52.48C23.552 451.584 0 428.032 0 398.848V52.736C0 23.552 23.552 0 52.48 0h343.04C424.448 0 448 23.552 448 52.736zm-72.96 242.688c0-82.432-36.864-149.248-36.864-149.248-36.864-27.648-71.936-26.88-71.936-26.88l-3.584 4.096c43.52 13.312 63.744 32.512 63.744 32.512-60.811-33.329-132.244-33.335-191.232-7.424-9.472 4.352-15.104 7.424-15.104 7.424s21.248-20.224 67.328-33.536l-2.56-3.072s-35.072-.768-71.936 26.88c0 0-36.864 66.816-36.864 149.248 0 0 21.504 37.12 78.08 38.912 0 0 9.472-11.52 17.152-21.248-32.512-9.728-44.8-30.208-44.8-30.208 3.766 2.636 9.976 6.053 10.496 6.4 43.21 24.198 104.588 32.126 159.744 8.96 8.96-3.328 18.944-8.192 29.44-15.104 0 0-12.8 20.992-46.336 30.464 7.68 9.728 16.896 20.736 16.896 20.736 56.576-1.792 78.336-38.912 78.336-38.912z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <footer
    className="footer mt-auto py-3 bg-dark text-white align-bottom"
  >
    <div
      className="footer footer-content mt-auto py-2 bg-dark text-white container"
    >
      <div
        className="footer-copyright text-center"
      >
        <div
          className="justify-content-md-center row row-cols-lg-3 row-cols-md-3 row-cols-sm-3 row-cols-3"
        >
          <div
            className="col"
            style={
              Object {
                "textAlign": "left",
              }
            }
          >
            © 2020 Copyright: CECEN
          </div>
          <div
            className="col"
            style={
              Object {
                "textAlign": "right",
              }
            }
          >
            Redes:
            <a
              className="btn btn-dark btn-sm"
              href="https://www.instagram.com/cecen.exactas/"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              target="_blank"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-instagram fa-w-14 fa-2x "
                data-icon="instagram"
                data-prefix="fab"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 448 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </a>
            <a
              className="btn btn-dark btn-sm"
              href="https://www.facebook.com/CECEN.UBA"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
              target="_blank"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-facebook fa-w-16 fa-2x "
                data-icon="facebook"
                data-prefix="fab"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </a>
          </div>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
